### PR TITLE
Move CTC meeting minutes into nodejs/CTC repo

### PR DESCRIPTION
**Pull request by jasnell**
Move the CTC meeting minutes into the nodejs/CTC repo to keep them out of the main nodejs/node repository.

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
Makes sense to me and is consistent with what we do for tsc minutes

**Review by williamkapke — APPROVED** [review]

**Comment by Trott** [comment]
+1 from me if we can get a bunch of other @nodejs/ctc folks to sign off on it too.

**Review by Trott — APPROVED** [review]

**Review by cjihrig — APPROVED** [review]

**Comment by jasnell** [comment]
@Trott: define "a bunch"? :-)

File: meetings/2015-10-28.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+* Shigeki Ohtsu (CTC)
+* Seth Thompson (observer)
+* Bert Belder (CTC)
+* Fedor Indutny (CTC)
+* Ben Noordhuis (CTC)
+* Colin Ihrig (CTC)
+
+## Agenda
+
+Extracted from **tsc-agenda** labelled issues and pull requests in the nodejs org prior to meeting.
```
Comment (thefourtheye) [inline meetings/2015-10-28.md:30]: `ctc-agenda`?

**Review by thefourtheye — APPROVED** [review]

**Comment by Trott** [comment]
> @Trott: define "a bunch"? :-)

http://www.merriam-webster.com/dictionary/bunch

Just kidding.

Counting you, we have 5 CTC members in favor. That seems to approximate "a bunch" so I'm good with it.

**Review by addaleax — APPROVED** [review]

**Comment by jasnell** [comment]
Awesome, pressing the green button now...

**Pull request merged**